tests/tensor/random/rewriting/test_basic.py

```diff
@@ -443,7 +446,8 @@ def test_DimShuffle_lift(ds_order, lifted, dist_op, dist_params, size, rtol):
         mode=no_mode,
     )
 
-    arg_values = [p.get_test_value() for p in f_inputs]
+    with pytest.warns(FutureWarning):
+        arg_values = [p.get_test_value() for p in f_inputs]
```

Comment:
Pass the test values to the test function. Otherwise this test will stop working once we remove test values